### PR TITLE
ntp test won't be running on tumbleweed

### PR DESCRIPTION
**Pull request by KatkaSuSe**
- Related ticket: https://progress.opensuse.org/issues/51695

### CODE REVIEW

**Comment by asdil12** [comment]
It doesn't make sense to run both `ntp_client` and `ntp` - especially as they would run in the same testsuite.
But you should also keep leap in mind - so better use `is_opensuse` here.

**Comment by Vogtinator** [comment]
It also fails on Leap, so this change is wrong.

The test itself should actually work just fine as starting ntpd stops chrony.

The failure is likely caused by using servers not actually reachable from the openSUSE network (`irc1.suse.de` and so on are internal). NTP does not adjust the time if the difference is above a certain threshold.

**Comment by Vogtinator** [comment]
Note that SLE 15 also uses chrony by default.

**Comment by asdil12** [comment]
Also instaling and starting `ntpd` will mess up the default that is later verified in the `ntp_client` test.

**Comment by asdil12** [comment]
I think this test shouldn't get executed within the same testsuite as `ntp_client`.

**Comment by Vogtinator** [comment]
IMO this test should be dropped or only ran on SLE 12.

**Comment by Vogtinator** [comment]
Ping on this topic.

File: lib/main_common.pm [hunk 1]
```diff
@@ -1613,7 +1613,7 @@ sub load_extra_tests_console {
     # sysauth test scenarios run in the console
     loadtest "sysauth/sssd" if get_var('SYSAUTHTEST') || is_sle('12-SP5+');
     loadtest 'console/timezone';
-    loadtest 'console/ntp';
+    loadtest 'console/ntp' unless is_tumbleweed;
```
Comment (baierjan) [inline lib/main_common.pm]: `is_sle('<15')`

**Comment by Vogtinator** [comment]
LGTM

**Pull request merged**